doc/miscellaneous.md

```diff
@@ -0,0 +1,40 @@
+# Miscellaneous
+
+## Dataset filter
+
+In order to filter datasets in scenario creation dialog (for scenario master only), there are two options:
+
+- API (Recommended)
+- Dataset filter (Deprecated)
+
+### API (Recommended)
+
+All dataset that are needed to be selectable in dataset list must be linked to the matching workspace.
+In the same way, datasets that are needed not to be visible must be unlinked from the workspace.
+
+This consists of using **_workspace/dataset_** `/link` - `/unlink` API endpoint
```

Comment:
not sure but it can be just one phrase:  _... unlinked from the workspace by using **_workspace/dataset_** ..._